packages/react-instantsearch-hooks-dom/src/widgets/Breadcrumb.tsx

```diff
@@ -6,11 +6,9 @@ import { Breadcrumb as BreadcrumbUiComponent } from '../ui/Breadcrumb';
 import type { BreadcrumbProps as BreadcrumbUiProps } from '../ui/Breadcrumb';
 import type { UseBreadcrumbProps } from 'react-instantsearch-hooks';
 
-export type BreadcrumbProps = Omit<
-  BreadcrumbUiProps,
-  'items' | 'hasItems' | 'createURL' | 'onNavigate' | 'translations'
-> &
-  Omit<UseBreadcrumbProps, 'separator'>;
+export type BreadcrumbProps = React.HTMLAttributes<HTMLDivElement> &
```

Comment:
Why don't we use this? It's a bit easier to read.

```suggestion
export type BreadcrumbProps = React.ComponentProps<'div'> &
```